src/VisualStudio/Core/Test.Next/Services/AssetProviderTests.cs

```diff
@@ -53,10 +53,10 @@ private static async Task TestAssetAsync(object data)
             var assetSource = new SimpleAssetSource(workspace.Services.GetService<ISerializerService>(), new Dictionary<Checksum, object>() { { checksum, data } });
 
             var provider = new AssetProvider(sessionId, storage, assetSource, remoteWorkspace.Services.GetService<ISerializerService>());
-            var stored = await provider.GetAssetAsync<object>(assetHint: AssetHint.None, checksum, CancellationToken.None);
+            var stored = await provider.GetAssetAsync<object>(AssetHint.FullLookupForTesting, checksum, CancellationToken.None);
```

Comment:
to ease with testing, tests can still say they want to just do a full tree lookup for a particular checksum, and the full tree will be walked.

Reply:
Usages of AssetHint.None in testing generally became this new 'do full walk' option.

AssetHint.None itself went away.